osu.Game.Rulesets.Osu/Skinning/Argon/ArgonMainCirclePiece.cs

```diff
@@ -138,53 +142,71 @@ private void updateStateTransforms(DrawableHitObject drawableHitObject, ArmedSta
                 switch (state)
                 {
                     case ArmedState.Hit:
+
+                        bool useFlash = config.Get<bool>(OsuSetting.HitLighting);
+                        var easingOutQuint = useFlash ? Easing.OutQuint : Easing.Out;
+
                         // Fade out time is at a maximum of 800. Must match `DrawableHitCircle`'s arbitrary lifetime spec.
-                        const double fade_out_time = 800;
+                        double fadeOutTime = useFlash ? 800 : 240;
 
-                        const double flash_in_duration = 150;
+                        double flashInDuration = fadeOutTime * (150 / 800.0);
```

Comment:
This calculation is weird. I think it would read better as

```suggestion
                        double flashInDuration = (fadeOutTime / 800) * 150;
```

Or maybe better to rewrite this whole change as:

```csharp
                        // Fade out time is at a maximum of 800. Must match `DrawableHitCircle`'s arbitrary lifetime spec.
                        const double fade_out_time = 800;
                        const double flash_in_duration = 150;

                        double animationTime = hitLighting ? 1 : 0.3f;

                        double fadeOutTime = animationTime * fade_out_time;
                        double flashInDuration = flash_in_duration * animationTime;
```

(please check this matches your current code, I haven't visually checked.)

Reply:
I'm un-resolving this because the code I proposed was not applied. Please don't self-resolve reviews; we'll do that for you.

Reply:
This was auto-resolved by GitHub, I used the `Commit suggestion` button, yup I'll double-check the visuals with the second code

Reply:
Seems fine, will commit